docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,274 @@
+    "\n",
+    "![Depiction of gate twirls](/images/run/error-mitigation-explanation/gate_twirls.png)\n",
+    "\n",
+    "The Qiskit Runtime options for Pauli twirling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.TwirlingOptions). The following options are notable:\n",
+    "\n",
+    "- `enable_gates`: Set this to `True` to enable Pauli twirling of two-qubit gates.\n",
+    "- `num_randomizations`: The number of circuit instances to draw from the ensemble of twirled circuits.\n",
+    "- `shots_per_randomization`: The number of shots to sample from each circuit instance.\n",
+    "\n",
+    "The following code cell shows how to enable Pauli twirling and set these options for the estimator."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "c3e12d44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(backend=backend)\n",
+    "estimator.options.twirling.enable_gates = True\n",
+    "estimator.options.twirling.num_randomizations = 32\n",
+    "estimator.options.twirling.shots_per_randomization = 100"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "ad9d9159-4277-4aec-bf26-da748e240f38",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "source": [
+    "## Twirled readout error extinction (TREX)\n",
+    "\n",
+    "[Twirled readout error extinction (TREX)](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.105.032620) mitigates the effect of measurement errors for the estimation of Pauli observable expectation values.\n",
+    " It is based on the notion of twirled measurements, which are accomplished by randomly substituting measurement gates by a sequence of (1) a Pauli X gate, (2) a measurement, and (3) classical bit flip. Just like in standard gate twirling, this sequence is equivalent to a plain measurement in the absence of noise, as depicted in the following diagram:\n",
+    "\n",
+    "![Depiction of measurement twirling](/images/run/error-mitigation-explanation/measurement_twirling.png)\n",
+    "\n",
+    "In the presence of readout error, measurement twirling has the effect of diagonalizing the readout-error transfer matrix, making it easy to invert. Estimating the readout-error transfer matrix requires executing additional calibration circuits, which introduces a small overhead.\n",
+    "\n",
+    "TREX can be enabled by setting `measure_mitigation` to `True` in version 2 of the [Qiskit Runtime resilience options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2). Options for measurement twirling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.MeasureNoiseLearningOptions). As in gate twirling, you can set the number of circuit randomizations and the number of shots per randomization.\n",
```

Comment:
```suggestion
    "TREX can be enabled by setting `measure_mitigation` to `True` in version 2 of the [Qiskit Runtime resilience options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2). Options for measurement twirling are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.MeasureNoiseLearningOptions). As with gate twirling, you can set the number of circuit randomizations and the number of shots per randomization.\n",
```

I think this is slightly better grammar, but @abbycross will know better 😅

Reply:
Done.